packages/terra-progress-bar/src/ProgressBar.scss

```diff
@@ -50,6 +58,14 @@
 
   // Firefox styles must be entirely separate as it doesn't work well with WebKit styles
   @-moz-document url-prefix('') {
+    .progress-bar-gradient {
+      &::-moz-progress-bar {
```

Comment:
Can we use a linear-gradient mixin here so that the code can avoid repetition. Refer to [this](https://gist.github.com/nickpiesco/4731166) source for example.